contracts/tunnel-consumer-ibc-hook/src/msg.rs

```diff
@@ -0,0 +1,30 @@
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use cosmwasm_std::Addr;
+use cw_band::tunnel::packet::TunnelPacket;
+
+#[cw_serde]
+pub struct InstantiateMsg {}
+
+#[cw_serde]
+pub enum ExecuteMsg {
+    ReceivePacket { packet: TunnelPacket },
+    AddSenders { msg: AddSendersMsg },
```

Comment:
Why not just
AddSenders { senders: Vec<Addr> }, ?
msg looks duplicated.

Reply:
We keep messages separate to allow for future modifications.